src/demo-app/accessibility/button/button-accessibility.html

```diff
@@ -0,0 +1,23 @@
+<div class="demo-button">
+  <section>
+    <button md-button>Check</button>
+    <button md-raised-button>raised</button>
+    <button md-fab>
+      <md-icon class="md-24">check</md-icon>
+    </button>
+    <button md-mini-fab>
+      <md-icon class="md-24">check</md-icon>
+    </button>
+  </section>
+
+  <section>
+    <a href="http://www.google.com" md-button color="primary">SEARCH</a>
```

Comment:
Avoid using ALL_CAPS directly because screen-readers will read them out as "S - E - A - R - C - H". 

This one should be, e.g., "Google search".